packages/palette-docs/plugins/gatsby-plugin-local-algolia-docsearch/gatsby-ssr.js

```diff
@@ -0,0 +1,30 @@
+const React = require("react")
+
+exports.onRenderBody = (
```

Comment:
👍

Reply:
I don't know how to make this plugin work with ES module syntax, but it isn't something that bothers me. If it bothers anyone else, maybe we could pair on it.

Reply:
ES module syntax doesn't work completely in gatsby yet. Caused a bit of stress trying to figure that out when i was working on this before because it _kinda_ worked at times, and was confusing.